packages/style/scss/controls/flat-select.scss

```diff
@@ -92,66 +84,37 @@
                 }
             }
         }
-
-        .icon {
-            margin-bottom: -2px;
-        }
     }
 
-    &.mod-btn-group .flat-select-option {
-        height: $button-height;
-        padding: $button-padding-y $button-padding-x;
-        font-weight: var(--main-font-bold);
+    &.mod-btn-group {
+        --gap: 0px;
+        border-radius: 8px;
 
-        font-size: $button-font-size;
-        text-transform: uppercase;
-        border-radius: 0;
+        .flat-select-option {
+            @extend .flat-select-option;
+            flex-grow: 1;
+            border-radius: 0;
 
-        &:focus {
-            z-index: 1;
-        }
-
-        &:first-child {
-            border-radius: $border-radius 0 0 $border-radius;
-        }
+            &:focus {
+                z-index: 1;
```

Comment:
It would be interesting to have global variables for plasma for the use of z-indexes, it can be done later

Reply:
I just left it there because idk why it's there xD